upf/upf_cli.c

```diff
@@ -254,6 +254,14 @@ upf_ueip_pool_add_del_command_fn (vlib_main_t * vm,
   nwi_name = upf_name_to_labels (nwi_s);
   vec_free (nwi_s);
 
+  if (strlen (nwi_name) > 64)
+    return clib_error_return (0,
+			      "NWI name(encoded) has to be shorter than 64 bytes");
+
+  if (strlen (name) > 64)
+    return clib_error_return (0,
+			      "UE IP pool name has to be shorter than 64 bytes");
+
```

Comment:
`upf_name_to_labels` returns a VPP vector which is not normally null-terminated
You need to use `vec_len` with vectors:
```suggestion
  if (vec_len (nwi_name) > 64)
    return clib_error_return (0,
			      "NWI name(encoded) has to be shorter than 64 bytes");

  if (vec_len (name) > 64)
    return clib_error_return (0,
			      "UE IP pool name has to be shorter than 64 bytes");

```